f0cal-tabcomplete-init.sh

```diff
@@ -0,0 +1,5 @@
+if ! grep -q "source $VIRTUAL_ENV/src/plugnparse/plugnparse/activate-argcomplete.sh" $VIRTUAL_ENV/bin/activate; then
```

Comment:
it might be nice to have the path to the arg complete activate script and a variable to avoid repeating it three times and to make it easier to change later if it needs to be?